freqtrade/commands/deploy_commands.py

```diff
@@ -63,6 +65,8 @@ def deploy_new_strategy(strategy_name: str, strategy_path: Path, subtemplate: st
                                                "sell_trend": sell_trend,
                                                "plot_config": plot_config,
                                                "additional_methods": additional_methods,
+                                               "trading_mode": 'futures' if leverage else 'spot',
+                                               "margin_mode": 'isolated' if leverage else ''
```

Comment:
i'd not take such assumptions ... (also see other comment about *_mode coming from the strategy - so this may not be necessary ... ).

By assuming that "Leverage == futures" - you're making it more difficult to support margin - which could also use leverage (but cannot rely on this automatism anymore).

Reply:
Ok, this can be removed now that I understand that normal strategies can also contain "enter_long" signals.